### ISSUE TITLE
Release 4.8.2 - RC 1 - Installation metrics

### ISSUE THREAD

**Issue by wazuhci** [wazuh/wazuh#25271]
### Packages tests metrics information
|||
| :--                              | :--                                                             |
| **Main release stage issue**     | #25268                                         |
| **Main packages metrics issue**  | #25270                                                   |
| **Version**                      | 4.8.2                                                   |
| **Release stage**                | RC 1                                                   |
| **Tag**                          | https://github.com/wazuh/wazuh/tree/v4.8.2-rc1          |

---

### Packages used
- Repository: `packages-dev.wazuh.com`
- Package path: `pre-release`
- Package revision: `1`

---

| Arquitecture | Build |
| :--         | :--   |
| AMD64        | https://ci.wazuh.info/job/Test_install_tier/984   |
| ARM64        | https://ci.wazuh.info/job/Test_install_tier/985   |
| I386         | https://ci.wazuh.info/job/Test_install_tier/986   |
| ARM32        | https://ci.wazuh.info/job/Test_install_tier/987   |

---

| System              | AMD64 | ARM64 | I386 | ARM32 |
| :--                 | :--:  | :--:  | :--: | :--:  |
| Amazon Linux 1      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Amazon Linux 2      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| CentOS 5            | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| CentOS 6            | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| CentOS 7            | :yellow_circle:    | :green_circle:    | :green_circle:   | :white_circle:    |
| CentOS 8            | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Debian 8            | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| Debian 9            | :green_circle:    | :green_circle:    | :green_circle:   | :green_circle:    |
| Debian 10           | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| Debian 11           | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| Debian 12           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Fedora 29           | :green_circle:    | :green_circle:    | :white_circle:   | :green_circle:    |
| Fedora 31           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Fedora 32           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Fedora 34           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Fedora 35           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Fedora 36           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| openSUSE Tumbleweed | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Oracle Linux 6      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Oracle Linux 7      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Oracle Linux 8      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Red Hat 6           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Red Hat 7           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Red Hat 8           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Red Hat 9           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Solaris 11          | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Solaris 10          | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Ubuntu Jammy       | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Ubuntu Bionic       | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Ubuntu Focal        | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Ubuntu Precise      | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Ubuntu Trusty       | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| Ubuntu Xenial       | :green_circle:    | :white_circle:    | :green_circle:   | :white_circle:    |
| Windows 7           | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 10          | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 11          | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 2012        | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 2016        | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 2019        | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| Windows 2022        | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| macOS 10.15         | :green_circle:    | :white_circle:    | :white_circle:   | :white_circle:    |
| macOS 12.01         | :white_circle:    | :green_circle:    | :white_circle:   | :white_circle:    |
| macOS 13.4          | :white_circle:    | :green_circle:    | :white_circle:   | :white_circle:    |

---

## macOS specific test

| System                  | Install* | Install** | TCP | UDP | Alerts | Logs | User | Group | Remove |
| :--                     | :--: | :--: | :--: | :--: | :--: | :--: | :--: | :--: | :--: |
| macOS 13.4  Ventura(intel64)     | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: |
| macOS 14  Sonoma(intel64)     | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: |
| macOS 13.4  Ventura(arm64)     | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: |
| macOS 14  Sonoma(arm64)     | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: | :white_circle: |

- \* Without environment variables
- \*\* With environment variables

---

Status legend:
:black_circle: Not finished: The tasks haven't been finished yet.
:white_circle: Skipped: The task has been deemed unnecessary for this stage.
:red_circle: Completed with failures: Some issues were raised here.
:yellow_circle: Completed with accepted known issues: Some issues were raised here, they were deemed non-stopper.
:green_circle: Completed: Task finished with success. Nothing to do here.

---

## Conclusion :yellow_circle:

The package installation tests were ran and no new errors were found. 

**Known issues**
- https://github.com/wazuh/wazuh/issues/24679

## Auditor's validation

In order to close and proceed with the release or the next candidate version, the following auditors must give the green light to this RC.

- [ ] @wazuh/devel-qa-release

---

**Comment by jseg380** [comment]
Running only automatic tests

**Comment by jseg380** [comment]
# Analysis report - AMD64

### Centos 5

Initial build: https://ci.wazuh.info/job/Test_install/166119/

<details>

<summary>Error logs</summary>

```
11:20:29  TASK [test-manager : Installation test manager (Pytest)] ***********************
11:20:29  fatal: [amd64-centos-7-base_0]: FAILED! => {
11:20:29      "changed": true,
11:20:29      "cmd": "/usr/testing/bin/pytest --tb=short install_manager_test.py --to_version 4.8.2 --os 'centos 7' -v",
11:20:29      "delta": "0:00:00.987121",
11:20:29      "end": "2024-08-20 09:20:29.629859",
11:20:29      "rc": 1,
11:20:29      "start": "2024-08-20 09:20:28.642738"
11:20:29  }
11:20:29  
11:20:29  STDOUT:
11:20:29  
11:20:29  ============================= test session starts ==============================
11:20:29  platform linux -- Python 3.8.2, pytest-5.4.1, py-1.11.0, pluggy-0.13.1 -- /usr/testing/bin/python3.8
11:20:29  cachedir: .pytest_cache
11:20:29  rootdir: /install_manager
11:20:29  collecting ... collected 14 items
11:20:29  
11:20:29  install_manager_test.py::test_check_version PASSED                       [  7%]
11:20:29  install_manager_test.py::test_check_process_clusterd PASSED              [ 14%]
11:20:29  install_manager_test.py::test_check_process_modulesd PASSED              [ 21%]
11:20:29  install_manager_test.py::test_check_process_monitord PASSED              [ 28%]
11:20:29  install_manager_test.py::test_check_process_logcollector PASSED          [ 35%]
11:20:29  install_manager_test.py::test_check_process_remoted PASSED               [ 42%]
11:20:29  install_manager_test.py::test_check_process_syscheckd PASSED             [ 50%]
11:20:29  install_manager_test.py::test_check_process_analysisd PASSED             [ 57%]
11:20:29  install_manager_test.py::test_check_process_maild PASSED                 [ 64%]
11:20:29  install_manager_test.py::test_check_process_execd PASSED                 [ 71%]
11:20:29  install_manager_test.py::test_check_process_wazuhdb PASSED               [ 78%]
11:20:29  install_manager_test.py::test_check_log_errors FAILED                    [ 85%]
11:20:29  install_manager_test.py::test_check_ossec_conf PASSED                    [ 92%]
11:20:29  install_manager_test.py::test_connection_manager PASSED                  [100%]
11:20:29  
11:20:29  =================================== FAILURES ===================================
11:20:29  ____________________________ test_check_log_errors _____________________________
11:20:29  install_manager_test.py:128: in test_check_log_errors
11:20:29      assert expected_error == True, assert_description['error']['exception_not_detected']
11:20:29  E   AssertionError: An error was found in the log
11:20:29  E   assert False == True
11:20:29  E     +False
11:20:29  E     -True
11:20:29  ----------------------------- Captured stdout call -----------------------------
11:20:29  2024/08/20 09:17:19 wazuh-modulesd:router: INFO: Loaded router module.
11:20:29  2024/08/20 09:17:19 wazuh-modulesd:content_manager: INFO: Loaded content_manager module.
11:20:29  2024/08/20 09:17:22 wazuh-csyslogd: INFO: Remote syslog server not configured. Clean exit.
11:20:29  2024/08/20 09:17:22 wazuh-dbd: INFO: Database not configured. Clean exit.
11:20:29  2024/08/20 09:17:22 wazuh-integratord: INFO: Remote integrations not configured. Clean exit.
11:20:29  2024/08/20 09:17:22 wazuh-agentlessd: INFO: Not configured. Exiting.
11:20:29  2024/08/20 09:17:22 wazuh-authd: INFO: Started (pid: 1160).
11:20:29  2024/08/20 09:17:22 wazuh-authd: INFO: Accepting connections on port 1515. No password required.
11:20:29  2024/08/20 09:17:23 wazuh-authd: INFO: Setting network timeout to 1.000000 sec.
11:20:29  2024/08/20 09:17:24 wazuh-db: INFO: Started (pid: 1176).
11:20:29  2024/08/20 09:17:25 wazuh-execd: INFO: Started (pid: 1190).
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: Started (pid: 1217).
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/bin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/boot', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/etc', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/sbin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/usr/bin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6003): Monitoring path: '/usr/sbin', with options 'size | permissions | owner | group | mtime | inode | hash_md5 | hash_sha1 | hash_sha256 | scheduled'.
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/mtab'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/hosts.deny'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/mail/statistics'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/random-seed'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/random.seed'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/adjtime'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/httpd/logs'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/utmpx'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/wtmpx'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/cups/certs'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/dumpdates'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6206): Ignore 'file' entry '/etc/svc/volatile'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6207): Ignore 'file' sregex '.log$|.swp$'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6004): No diff for file: '/etc/ssl/private.key'
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6000): Starting daemon...
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6010): File integrity monitoring scan frequency: 43200 seconds
11:20:29  2024/08/20 09:17:26 wazuh-syscheckd: INFO: (6008): File integrity monitoring scan started.
11:20:29  2024/08/20 09:17:26 rootcheck: INFO: Starting rootcheck scan.
11:20:29  2024/08/20 09:17:27 wazuh-remoted: INFO: Started (pid: 1234). Listening on port 1514/TCP (secure).
11:20:29  2024/08/20 09:17:27 wazuh-remoted: INFO: Cannot find 'queue/db/wdb'. Waiting 1 seconds to reconnect.
11:20:29  2024/08/20 09:17:28 wazuh-analysisd: INFO: Total rules enabled: '6786'
11:20:29  2024/08/20 09:17:28 wazuh-analysisd: INFO: Started (pid: 1205).
11:20:29  2024/08/20 09:17:28 wazuh-analysisd: INFO: Cannot find 'queue/db/wdb'. Waiting 1 seconds to reconnect.
11:20:29  2024/08/20 09:17:28 wazuh-logcollector: INFO: Monitoring output of command(360): df -P
11:20:29  2024/08/20 09:17:28 wazuh-logcollector: INFO: Monitoring full output of command(360): netstat -tulpn | sed 's/\([[:alnum:]]\+\)\ \+[[:digit:]]\+\ \+[[:digit:]]\+\ \+\(.*\):\([[:digit:]]*\)\ \+\([0-9\.\:\*]\+\).\+\ \([[:digit:]]*\/[[:alnum:]\-]*\).*/\1 \2 == \3 == \4 \5/' | sort -k 4 -g | sed 's/ == \(.*\) ==/:\1/' | sed 1,2d
11:20:29  2024/08/20 09:17:28 wazuh-logcollector: INFO: Monitoring full output of command(360): last -n 20
11:20:29  2024/08/20 09:17:28 wazuh-logcollector: INFO: (1950): Analyzing file: '/var/ossec/logs/active-responses.log'.
11:20:29  2024/08/20 09:17:28 wazuh-logcollector: INFO: Started (pid: 1247).
11:20:29  2024/08/20 09:17:28 wazuh-remoted: INFO: Cannot find 'queue/db/wdb'. Waiting 2 seconds to reconnect.
11:20:29  2024/08/20 09:17:29 wazuh-analysisd: INFO: Cannot find 'queue/db/wdb'. Waiting 2 seconds to reconnect.
11:20:29  2024/08/20 09:17:29 wazuh-monitord: INFO: Started (pid: 1268).
11:20:29  2024/08/20 09:17:30 wazuh-remoted: INFO: Cannot find 'queue/db/wdb'. Waiting 3 seconds to reconnect.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:router: INFO: Loaded router module.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:content_manager: INFO: Loaded content_manager module.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd: INFO: Started (pid: 1301).
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:control: INFO: Starting control thread.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:download: INFO: Module started.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:database: INFO: Module started.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:content_manager: INFO: Starting content_manager module.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:router: INFO: Starting router module.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:vulnerability-scanner: INFO: Starting vulnerability_scanner module.
11:20:29  2024/08/20 09:17:30 sca: INFO: Module started.
11:20:29  2024/08/20 09:17:30 sca: INFO: Loaded policy '/var/ossec/ruleset/sca/cis_centos7_linux.yml'
11:20:29  2024/08/20 09:17:30 sca: INFO: Starting Security Configuration Assessment scan.
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:osquery: INFO: Module disabled. Exiting...
11:20:29  2024/08/20 09:17:30 wazuh-modulesd:ciscat: INFO: Module disabled. Exiting...
11:20:29  2024/08/20 09:17:31 wazuh-modulesd:agent-upgrade: INFO: (8153): Module Agent Upgrade started.
11:20:29  2024/08/20 09:17:31 wazuh-modulesd:task-manager: INFO: (8200): Module Task Manager started.
11:20:29  2024/08/20 09:17:31 sca: INFO: Starting evaluation of policy: '/var/ossec/ruleset/sca/cis_centos7_linux.yml'
11:20:29  2024/08/20 09:17:31 wazuh-db: INFO: Created Global database backup "backup/db/global.db-backup-2024-08-20-09:17:31.gz"
11:20:29  2024/08/20 09:17:31 wazuh-modulesd:syscollector: INFO: Module started.
11:20:29  2024/08/20 09:17:31 wazuh-modulesd:syscollector: INFO: Starting evaluation.
11:20:29  2024/08/20 09:17:31 indexer-connector: WARNING: No username and password found in the keystore, using default values.
11:20:29  2024/08/20 09:17:32 wazuh-modulesd:syscollector: INFO: Evaluation finished.
11:20:29  2024/08/20 09:17:32 wazuh-analysisd: INFO: EPS limit disabled
11:20:29  2024/08/20 09:17:32 wazuh-analysisd: INFO: (7200): Logtest started
11:20:29  2024/08/20 09:17:32 indexer-connector: WARNING: IndexerConnector initialization failed for index 'wazuh-states-vulnerabilities-ip-172-31-58-165.ec2.internal', retrying until the connection is successful.
11:20:29  2024/08/20 09:17:32 wazuh-modulesd:vulnerability-scanner: INFO: Starting database file decompression.
11:20:29  2024/08/20 09:17:33 wazuh-syscheckd: INFO: (6009): File integrity monitoring scan ended.
11:20:29  2024/08/20 09:17:33 wazuh-syscheckd: INFO: FIM sync module started.
11:20:29  2024/08/20 09:17:33 wazuh-remoted: ERROR: Unable to connect to socket 'queue/db/wdb'.
11:20:29  2024/08/20 09:17:33 wazuh-remoted: ERROR: Unable to connect to socket 'queue/db/wdb'.
11:20:29  2024/08/20 09:17:33 wazuh-remoted: ERROR: Error querying Wazuh DB to get agent's groups.
11:20:29  2024/08/20 09:17:33 wazuh-remoted: INFO: (1410): Reading authentication keys file.
11:20:29  2024/08/20 09:18:26 rootcheck: INFO: Ending rootcheck scan.
11:20:29  2024/08/20 09:18:32 sca: INFO: Evaluation finished for policy '/var/ossec/ruleset/sca/cis_centos7_linux.yml'
11:20:29  2024/08/20 09:18:32 sca: INFO: Security Configuration Assessment scan finished. Duration: 62 seconds.
11:20:29  2024/08/20 09:18:38 wazuh-authd: INFO: New connection from 172.31.24.20
11:20:29  2024/08/20 09:18:38 wazuh-authd: INFO: Received request for a new agent (ip-172-31-24-20.ec2.internal) from: 172.31.24.20
11:20:29  2024/08/20 09:18:38 wazuh-authd: INFO: Agent key generated for 'ip-172-31-24-20.ec2.internal' (requested by any)
11:20:29  2024/08/20 09:18:43 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
11:20:29  2024/08/20 09:18:43 wazuh-remoted: INFO: (1410): Reading authentication keys file.
11:20:29  
11:20:29  =========================== short test summary info ============================
11:20:29  FAILED install_manager_test.py::test_check_log_errors - AssertionError: An er...
11:20:29  ========================= 1 failed, 13 passed in 0.20s =========================
11:20:29  
11:20:29  
11:20:29  MSG:
11:20:29  
11:20:29  non-zero return code
```

</details>

- Replayed: https://ci.wazuh.info/job/Test_install/166177/ :black_circle:

**Comment by santipadilla** [comment]
LGTM

**Comment by Rebits** [comment]
### Centos 7

This failure is produced due to a known issue reported and fixed for 4.9.0 https://github.com/wazuh/wazuh/issues/24679

I have included it in the conclusion


***

LGTM
